src/libs/auth.ts

```diff
@@ -34,52 +32,12 @@ import {
 import * as Utils from 'src/libs/utils';
 import { v4 as uuid } from 'uuid';
 
-export interface OidcUser extends User {
-  profile: B2cIdTokenClaims;
-}
-
-// Our config for b2C claims are defined here: https://github.com/broadinstitute/terraform-ap-deployments/tree/master/azure/b2c/policies
-// The standard b2C claims are defined here: https://learn.microsoft.com/en-us/azure/active-directory/develop/id-token-claims-reference
-export interface B2cIdTokenClaims extends IdTokenClaims {
-  email_verified?: boolean;
-  idp?: string;
-  idp_access_token?: string;
-  tid?: string;
-  ver?: string;
-}
-
-export const getOidcConfig = () => {
-  const metadata = {
-    authorization_endpoint: `${getConfig().orchestrationUrlRoot}/oauth2/authorize`,
-    token_endpoint: `${getConfig().orchestrationUrlRoot}/oauth2/token`,
-  };
-  return {
-    authority: `${getConfig().orchestrationUrlRoot}/oauth2/authorize`,
-    // The clientId from oidcStore.config is not undefined, setup in initializeClientId on appLoad
-    client_id: oidcStore.get().config.clientId!,
-    popup_redirect_uri: `${window.origin}/redirect-from-oauth`,
-    silent_redirect_uri: `${window.origin}/redirect-from-oauth-silent`,
-    metadata,
-    prompt: 'consent login',
-    scope: 'openid email profile',
-    stateStore: new WebStorageStateStore({ store: getLocalStorage() }),
-    userStore: new WebStorageStateStore({ store: getLocalStorage() }),
-    automaticSilentRenew: true,
-    // Leo's setCookie interval is currently 5 min, set refresh auth then 5 min 30 seconds to guarantee that setCookie's token won't expire between 2 setCookie api calls
-    accessTokenExpiringNotificationTimeInSeconds: 330,
-    includeIdTokenInSilentRenew: true,
-    extraQueryParams: { access_type: 'offline' },
-    redirect_uri: '', // this field is not being used currently, but is expected from UserManager
-  };
-};
-
-const getAuthInstance = (): AuthContextProps => {
-  const authContext: AuthContextProps | undefined = oidcStore.get().authContext;
-  if (authContext === undefined) {
-    console.error('getAuthInstance must not be called before authContext is initialized.');
-    throw new Error('Error initializing authentication.');
+export const getAuthToken = (): string | undefined => {
+  const oidcUser: OidcUser | undefined = oidcStore.get().user;
+  if (oidcUser !== undefined) {
+    return oidcUser.access_token;
   }
-  return authContext;
+  return undefined;
```

Comment:
It looks like this could possibly be simplified to `return oidcUser?.access_token`.